src/tools/weave/Cmd_GenCodeSigningCert.cpp

```diff
@@ -168,10 +175,17 @@ bool Cmd_GenCodeSigningCert(int argc, char *argv[])
         ExitNow(res = false);
     }
 
-    if (gNewCertKeyFileName == NULL)
+    if (gNewCertKeyFileName == NULL && gNewCertPubKeyFileName == NULL)
+    {
+        fprintf(stderr, "Please use the --key option to specify the public/private key file for the\n"
+                        "new certificate or use the --pubkey option to specify the public key file\n");
+        ExitNow(res = false);
+    }
+
+    if (gNewCertKeyFileName != NULL && gNewCertPubKeyFileName != NULL)
     {
         fprintf(stderr, "Please use the --key option to specify the public/private key file for the\n"
-                        "new certificate.\n");
+                        "new certificate or use the --pubkey option to specify the public key file\n");
```

Comment:
I think this should say something like "Please specify only one of --key or --pubkey".

Reply:
Agree I'll make the change.